<launch>

    <node pkg="love_maker_2000" type="cylinder_approaching_point_calculator.py" name="cylinder_approaching_point_calculator">
    </node>

</launch>